etc/Language/Razor_lang.enu

```diff
@@ -807,4 +807,6 @@ InputBox::ok=&OK
 2104=Masteries
 2105=Waypoint: Next
 2106=Waypoint: Previous
-2107=Waypoint: Hide
\ No newline at end of file
+2107=Waypoint: Hide
+2108=Next Non-Friendly Target
+2109=Prev Non-Friendly Target
```

Comment:
How is the other localization usually handled?

Reply:
Which other localization?

Reply:
I just meant the various other files in this directory. Seems like those are from before and not really updated so I guess the answer is nothing haha